README.md

```diff
@@ -200,3 +200,22 @@ The deployments of Terra Data Repository are:
 - [Production](https://data.terra.bio/)
 - [Development](https://jade.datarepo-dev.broadinstitute.org/)
 - [Development - swagger page](https://jade.datarepo-dev.broadinstitute.org/swagger-ui.html)
+
+### Running Sonar locally
+
+[Sonar](https://www.sonarqube.org) is a static analysis code that scans code for a wide
+range of issues, including maintainability and possible bugs. If you get a build failure due to
+SonarQube and want to debug the problem locally, you need to get the sonar token from vault
+before running the gradle task.
+
+```shell
+export SONAR_TOKEN=$(vault read -field=sonar_token secret/secops/ci/sonarcloud/data-repo)
+./gradlew sonar
+```
+
+Unlike SourceClear, running this task produces no output unless your project has errors. To always
+generate a report, run using `--info`:
```

Comment:
TDR doesn't use source clear yet.
```suggestion
Running this task produces no output unless your project has errors. To always
generate a report, run using `--info`:
```

Reply:
Ah, thanks, this is leftover from another repo.